selector.go

```diff
@@ -106,24 +107,35 @@ func FilterIn(values ...string) *SimpleFilter {
 func FilterNotIn(values ...string) *SimpleFilter {
 	fs := make([]*SimpleFilter, 0, len(values))
 	for _, v := range values {
-		fs = append(fs, FilterNE(v))
+		f := FilterNE(v)
+		fs = append(fs, f)
 	}
 	return FilterAND(fs...)
 }
 
 // FilterOR returns OR combination of filters.
 func FilterOR(fs ...*SimpleFilter) *SimpleFilter {
+	args := make([]SimpleFilter, 0, len(fs))
```

Comment:
I would rather think about whether we want to work with pointers at all. Especially since slice of `SimpleFilter` is now slice of structs instead of slice of pointers. 

I think @fyrchik review will be useful on this.

Reply:
I am not sure that it is necessary to change so radically now.
My change was needed to get rid of the pointers in the slice. In case of abandonment of pointers in the structure itself, it can have a negative impact on the whole.

Waiting for @fyrchik